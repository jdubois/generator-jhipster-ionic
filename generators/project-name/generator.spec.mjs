import { expect } from 'expect';

import { helpers, lookups } from '#test-utils';

describe('SubGenerator project-name of ionic JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create('jhipster:project-name')
        .withOptions({
          reproducible: true,
          defaults: true,
          blueprint: 'ionic',
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
