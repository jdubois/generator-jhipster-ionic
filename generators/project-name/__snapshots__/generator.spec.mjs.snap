// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator project-name of ionic JHipster blueprint run should succeed 1`] = `
{
  ".yo-rc.json": {
    "stateCleared": "modified",
  },
}
`;
